dev/Sumangrewal/Data_vizualization.ipynb

```diff
@@ -0,0 +1,1063 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 123,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import pandas as pd\n",
+    "import seaborn as sns\n",
+    "from sklearn.preprocessing import StandardScaler, LabelEncoder\n",
+    "from program import read_data, split_data\n",
+    "from sklearn.linear_model import LogisticRegression\n",
+    "from sklearn.metrics import confusion_matrix,accuracy_score\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Reading the data using the function read_data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 124,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset = read_data(r'C:\\Users\\HP\\Desktop\\PRESC\\datasets\\winequality.csv')"
```

Comment:
This needs to be a relative path so that anyone can run the notebook from the repo.

Reply:
@dzeber path changed! Thankyou!